.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,16 @@
 <!-- Thank you for reporting! -->
-
 <!-- Please go to the forum first for support with crashes and issues. -->
 <!-- http://www.minecraftforge.net/forum/forum/18-support-bug-reports/ -->
+
+**Minecraft Version:** {mc version}
+
+**Forge Version:** {forge version}
```

Comment:
You might need a "**not** the Minecraft version, **DO NOT** specify 'latest'/'recommended'" here.

Reply:
I don't think it's necessary to mention *not* the Minecraft version as it's mentioned in it's own placeholder directly above.